### PR TITLE
Simulator refactor

### PR DESCRIPTION
**Pull request by ZenoTan**
go mod tidy
refactor simulator more cases
report artifact link

### CODE REVIEW

**Comment by mahjonp** [comment]
After https://github.com/lhy1024/bench/pull/39 was merged, you need to update the workload configuration:

https://github.com/lhy1024/bench/blob/46aeeb2844f1b8ef9e193b6599db8e5d6bf6274d/.workloads/simulator.json#L29-L30

**Comment by mahjonp** [comment]
Rest LGTM

File: bench/cases.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ type Benches struct {
 func NewBenches(cluster *Cluster) *Benches {
 	caseMap := make(map[string]*Case)
 	caseMap["scale-out"] = CreateScaleOutCase(cluster)
-	caseMap["sim"] = CreateSimulatorCase(cluster)
+	caseMap["sim-import"] = CreateSimulatorCase(cluster, "import")
```
Comment (lhy1024) [inline bench/cases.go:15]: we maybe provide a method to show support benches

**Review by lhy1024 — APPROVED** [review]
LGTM.

**Pull request merged**